File: C++/Daily Flash/Week2/Day1_Jan20/Code/program3.cpp
/*
 * Program 3 : Write a Program that accepts a n inte ger from user and prints
whether that number is odd or even .
Input: 2
Output: 2 is an Even Number
*
*/

#include <iostream>

int main() {
	int num;
	std::cout << "Enter num: " ;
	std::cin >> num;

	if(num%2 == 0)
		std::cout << num <<" is an Even number" << std::endl;
	else
		std::cout << num <<" is an Odd number" << std::endl;
	return 0;
}

File: JAVA/Daily Flash Codes/Week2/Day1_Jan20/code/program3.java
/*
 * Program 3 : Write a Program that accepts a n inte ger from user and prints
whether that number is odd or even .
Input: 2
Output: 2 is an Even Number
*
*/
import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		System.out.println("Enter integer: ");
		int num = Integer.parseInt(in.readLine());
		if(num%2 == 0)
			System.out.println(num +" is an Even number");
		else
			System.out.println(num +" is an Odd number");


	}


}
